api/configuration/appConfig.ts

```diff
@@ -19,6 +19,7 @@ import {
   SERVICES_CCD_CASE_ASSIGNMENT_API_PATH,
   SERVICES_CCD_COMPONENT_API_PATH,
   SERVICES_CCD_DATA_STORE_API_PATH,
+  SERVICES_COMMONRD_API_URL,
```

Comment:
seems it's duplicated with line 33 SERVICES_PRD_COMMONDATA_API, can that reuse that env variable?

Reply:
Hi @rayliangatos - I'll have a look. I guess this is because the Hearings feature changes are already in `master`

Reply:
yes the initial hearing is on master